src/cf-atomic-component/src/components/AtomicComponent.js

```diff
@@ -193,11 +193,14 @@ assign( AtomicComponent.prototype, Events, classList, {
     this.undelegateEvents();
     this._delegate = new Delegate( this.element );
     for ( key in events ) {
-      method = events[key];
-      if ( isFunction( this[method] ) ) method = this[method];
-      if ( !method ) continue;
-      match = key.match( delegateEventSplitter );
-      this.delegate( match[1], match[2], bind( method, this ) );
+      if ( {}.hasOwnProperty.call( events, key ) ) {
```

Comment:
`if ( events.hasOwnProperty( key ) ) {` is probably clearer and equivalent?

Reply:
Also, I left this one in https://github.com/cfpb/capital-framework/pull/838 since I'm not totally certain the intention isn't to have `key` retrieved from the parents due to the inheritance structure of AtomicComponent.

Reply:
Is it equivalent? I was following the examples in the [ESLint documentation for guard-for-in](https://eslint.org/docs/rules/guard-for-in), but if you think this is better, I'm fine with that.

I don't follow your second comment. I also don't really understand AtomicComponent, so... /shrug

Reply:
> Is it equivalent? I was following the examples in the ESLint documentation for guard-for-in, but if you think this is better, I'm fine with that.

I'm asking [here](https://github.com/eslint/eslint/commit/551335eedf62261299af7aafdfc896b3aef8de67), but as you have it works obviously, so no prob.

> I don't follow your second comment. I also don't really understand AtomicComponent, 

Let's give it a shot.